Makefile

```diff
@@ -100,7 +100,7 @@ distclean: sub-distclean this-distclean
 .PHONY: this-config this-build this-only this-test-suite this-test-suite-stdlib this-distclean this-clean
 
 this-config:: __always__
-	@command -v coqc >/dev/null || exit 1
+	@command -v coqc >/dev/null || exit 0  # Maybe on Rocq (without coq shim), nothing to do anyway
```

Comment:
does this actually skip the rest of the rule? I don't think you're using ONESHELL

Reply:
Indeed, that's broken. Should be fixed by https://github.com/math-comp/hierarchy-builder/pull/537
That being said, it's independent from the error we observe in [#Rocq devs &amp; plugin devs > HB doesn't seem to like the Rocq dev layout](https://rocq-prover.zulipchat.com/#narrow/channel/237656-Rocq-devs-.26-plugin-devs/topic/HB.20doesn't.20seem.20to.20like.20the.20Rocq.20dev.20layout) making it impossible to compile HB when it's already installed